pypeit/core/qa.py

```diff
@@ -45,7 +45,7 @@ def set_qa_filename(root, method, det=None, slit=None, prefix=None, out_dir=None
         outfile = 'QA/PNGs/Slit_Profile_{:s}_'.format(root)
     elif method == 'arc_fit_qa':
 #        outfile = 'QA/PNGs/Arc_1dfit_{:s}_S{:04d}.png'.format(root, slit)
-        outfile = 'PNGs/Arc_1dfit_{:s}_S{:04d}.png'.format(root, slit)
+        outfile = 'PNGs/Arc_1dfit_{:s}_S{:04d}.png'.format(root, slit) if slit is not None else 'PNGs/Arc_1dfit_{:s}.pdf'.format(root)
```

Comment:
Just to check: Does this affect the output QA html files that are written?

Reply:
Thanks @kbwestfall  for looking into these changes, and raising this point. I haven't fully tested this yet. I was going to do it today.
But just to give you some context, the reason I have done that is that I am hoping to share these plots with KOA whenever we share our reduced ADAP data and I though they needed to be more easily shareable.

Reply:
Thanks for this comment! I decided to revert these changes. As you guessed this affected the QA html, i.e., the wavelength fits were not saved in the html file. I decided that for KOA probably it will be easier to create html files for only the wavelength calibration (these are the only QA that I think it's important to share), but I realize I will need some help with that.